src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -315,6 +346,46 @@ public static void ExecutingAction(this ILogger logger, ActionDescriptor action)
             _actionExecuting(logger, action.DisplayName, null);
         }
 
+        public static void ExecutingAuthorizationFilters(this ILogger logger, IEnumerable<IFilterMetadata> filters)
```

Comment:
what about validating that filters isn't null first.

Reply:
This applies to all the similar methods really.

Reply:
Sure 👍